### ISSUE TITLE
Typescript error in `nglTypes.d.ts` when using `typescript@latest`

### ISSUE THREAD

**Issue by superjax** [gky360/react-ngl#10]
I'm trying to include `react-ngl` in a React Typescript project and I'm getting this:

```TypeScript error in /home/jjackson/Code/origami/ori_viz/node_modules/react-ngl/dist/utils/ngl/nglTypes.d.ts(3,1):
Declaration or statement expected.  TS1128

    1 | export * from 'ngl';
    2 | export { Matrix3, Matrix4, Vector2, Vector3, Box3, Quaternion, Euler, Plane, Color, } from 'three';
  > 3 | export type { StageParameters } from 'ngl/declarations/stage/stage';
      | ^
    4 | export type { ComponentParameters } from 'ngl/declarations/component/component';
    5 | export type { StructureRepresentationType } from 'ngl/declarations/component/structure-component';
    6 | export type { SurfaceRepresentationType } from 'ngl/declarations/component/surface-component';
```
If I set my typsecript version to 3.8, then the problem goes away.

**Comment by gky360** [comment]
Thank you for reporting an issue!
I'll investigate the problem this weekend.

**Comment by gky360** [comment]
I've fixed the problem and [released v0.2.0](https://github.com/gky360/react-ngl/releases).
Please check the v0.2.0 and kindly reopen the issue if you find the problem is not resolved.

**Comment by superjax** [comment]
Great!  Thanks
